components/user_settings/notifications/notification_schedule_setting.scss

```diff
@@ -0,0 +1,194 @@
+.form-switch {
+    display: flex;
+    align-items: center;
+    border: none;
+    margin: 0 0 0.8rem;
+    color: inherit;
+    font-size: inherit;
+    font-weight: 600;
+
+    .switch {
+        position: relative;
+        display: inline-block;
+        width: 48px;
+        height: 28px;
+        margin-right: 8px;
+
+        input {
+            width: 0;
+            height: 0;
+            opacity: 0;
+        }
+
+        .slider {
+            position: absolute;
+            top: 0;
+            right: 0;
+            bottom: 0;
+            left: 0;
+            background-color: #ccc;
```

Comment:
Update background color to use theme variable `background-color: var(--center-channel-color-24);`